internal/postprocessor/config.yaml

```diff
@@ -44,6 +44,7 @@ modules:
   - datastream
   - deploy
   - dialogflow
+  - discoveryengine
```

Comment:
@codyoss Was the omission of the module configuration intentional for `discoveryengine/v1beta`? Assuming not, adding it now.

Reply:
I don't think so.